modules/alloy/src/main/ts/ephox/alloy/api/component/GuiFactory.ts

```diff
@@ -90,16 +90,16 @@ const external = (spec: ExternalElement): PremadeSpec => {
 // There are other solutions than this ... not sure if they are going to have better performance, though
 const uids = Tagger.generate;
 
-const isSimpleOrSketchSpec = (spec: AlloySpec): spec is SimpleOrSketchSpec =>
+const isSketchSpec = (spec: AlloySpec): spec is SketchSpec =>
   Obj.has(spec as SimpleOrSketchSpec, 'uid');
 
 // INVESTIGATE: A better way to provide 'meta-specs'
 const build = (spec: AlloySpec): AlloyComponent => GuiTypes.getPremade(spec).getOrThunk(() => {
   // EFFICIENCY: Consider not merging here, and passing uid through separately
-  const userSpecWithUid = isSimpleOrSketchSpec(spec) ? spec : {
+  const userSpecWithUid = isSketchSpec(spec) ? spec : {
```

Comment:
This was incorrect as a sketch spec is the only thing that's guarnteed to have a `uid`.